week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,17 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  price = "$ " + (price * 1.4).toFixed(2);
```

Comment:
If you read the exercise description carefully, it says that the business is adding a 1% fee to **all** foreign transactions, not just Brazilian ones. How could you change your `convertToUSD` function to handle this?

Once you've done that, have a look at both of your functions. Do they have similar lines? How could we change them to reduce this? (May be google the "DRY principle")